src/main/java/org/broadleafcommerce/menu/domain/MenuImpl.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.broadleafcommerce.menu.domain;
 
+import jakarta.persistence.Index;
```

Comment:
Maybe move this import to group together with other jakarta imports.

Reply:
Sure!
didn't see this, thanks

Reply:
The[ import order](https://github.com/BroadleafCommerce/CodeStyle/blob/master/src/main/resources/broadleaf-java.importorder) is not correct. Move these back